skymap_scanner/recos/millipede_original.py

```diff
@@ -44,25 +43,32 @@ class MillipedeOriginal(RecoInterface):
     # Constants ########################################################
     pulsesName = cfg.INPUT_PULSES_NAME
     pulsesName_cleaned = pulsesName+'LatePulseCleaned'
-    SPEScale = 0.99
+
+    SPEScale = 0.99 # DOM efficiency
```

Comment:
Would it make sense to move all these variables to `__init__` and make `def exclusions` a bound method?

Reply:
I agree on the general idea of getting rid of `ClassName.constant` references.

I would still leave proper "constants" at the class level (we can still access them through `self` in bound/class methods after all).

I would keep `exclusions()` a class method as long as its parameters do not need to be defined at runtime, but of course we can change this in the future.